package.json

```diff
@@ -3,13 +3,14 @@
 	"version": "0.1.0",
 	"private": true,
 	"scripts": {
-		"dev": "API_URL=http://localhost:3000/api next dev",
+		"dev": "cross-env API_URL=http://localhost:3000/api next dev",
 		"build": "next build",
 		"start": "next start"
 	},
 	"dependencies": {
 		"axios": "^0.20.0",
 		"cookies": "^0.8.0",
+		"cross-env": "^7.0.3",
```

Comment:
This should be saved as a devDependency because it's not used when running in production via `yarn start`. :)

Reply:
Changed it 😉